pylint_odoo/examples/.jslintrc

```diff
@@ -144,12 +146,15 @@
     "arrow-spacing": "off",
     "accessor-pairs": "error",
     "block-scoped-var": "off",
-    "block-spacing": "off",
-    "brace-style": "error",
+    "block-spacing": ["error", "never"],
+    "brace-style": "warn",
```

Comment:
Personally, I think this must be kept as an error in order to encourage the programmers to follow brace-style